<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindowClass</class>
 <widget class="QMainWindow" name="CMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>介子缺陷检测软件_V1.0.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="CMainWindow.qrc">
    <normaloff>:/CMainWindow/Resources/app.ico</normaloff>:/CMainWindow/Resources/app.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_First">
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="title">
       <string>一工位</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_FirstTime">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>51</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 255);
color: rgb(85, 0, 255);</string>
         </property>
         <property name="text">
          <string>耗时：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_FirstIndex">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>51</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 255);
color: rgb(85, 0, 255);</string>
         </property>
         <property name="text">
          <string>序号：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_FirstResult">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>51</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 255);
color: rgb(85, 0, 255);
image: url(:/CMainWindow/Resources/OK.svg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="MyLabel" name="label_FirstImage">
         <property name="styleSheet">
          <string notr="true">background-image: url(:/CMainWindow/Resources/background.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_Second">
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="title">
       <string>二工位</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_SecondTime">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>51</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 255);
color: rgb(85, 0, 255);</string>
         </property>
         <property name="text">
          <string>耗时：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_SecondIndex">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>51</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 255);
color: rgb(85, 0, 255);</string>
         </property>
         <property name="text">
          <string>序号：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_SecondResult">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>51</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 255);
color: rgb(85, 0, 255);
image: url(:/CMainWindow/Resources/NG.svg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="MyLabel" name="label_SecondImage">
         <property name="styleSheet">
          <string notr="true">background-image: url(:/CMainWindow/Resources/background.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Start"/>
   <addaction name="action_Stop"/>
   <addaction name="separator"/>
   <addaction name="action_Setting"/>
   <addaction name="action_Reset"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Start">
   <property name="icon">
    <iconset resource="CMainWindow.qrc">
     <normaloff>:/CMainWindow/Resources/start.png</normaloff>:/CMainWindow/Resources/start.png</iconset>
   </property>
   <property name="text">
    <string>开始运行</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="icon">
    <iconset resource="CMainWindow.qrc">
     <normaloff>:/CMainWindow/Resources/stop.png</normaloff>:/CMainWindow/Resources/stop.png</iconset>
   </property>
   <property name="text">
    <string>停止运行</string>
   </property>
  </action>
  <action name="action_Setting">
   <property name="icon">
    <iconset resource="CMainWindow.qrc">
     <normaloff>:/CMainWindow/Resources/settings.png</normaloff>:/CMainWindow/Resources/settings.png</iconset>
   </property>
   <property name="text">
    <string>参数设置</string>
   </property>
  </action>
  <action name="action_Reset">
   <property name="icon">
    <iconset resource="CMainWindow.qrc">
     <normaloff>:/CMainWindow/Resources/reset.png</normaloff>:/CMainWindow/Resources/reset.png</iconset>
   </property>
   <property name="text">
    <string>统计重置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="CMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
